ambry-api/src/main/java/com/github/ambry/clustermap/VcrClusterAgentFactory.java

```diff
@@ -18,12 +18,15 @@
 
 
 /**
- * A factory to create {@link ClusterSpectator} object.
+ * A factory interface to get {@link VcrClusterParticipant} and {@link VcrClusterSpectator}.
  */
-public class HelixClusterSpectatorFactory implements ClusterSpectatorFactory {
+public interface VcrClusterAgentFactory {
```

Comment:
nit: We may want to make agent plural to match `ClusterAgentsFactory`. Unless you think the way it is sounds better

Reply:
done